File: segitiga full.js
// Pola 1: Piramida (Loop for)
let rows = 5;
for (let i = 1; i <= rows; i++) {
    let line = "";
    for (let j = 1; j <= rows - i; j++) {
        line += " ";
    }
    for (let k = 1; k <= 2 * i - 1; k++) {
        line += "*";
    }
    console.log(line);
}
console.log(""); 


let rows2 = 5;
while (rows2 >= 1) {
    console.log("*".repeat(rows2));
    rows2--;
}
console.log("");

let rows3 = 5;
let i = 1;
do {
    console.log(" ".repeat(rows3 - i) + "*".repeat(i));
    i++;
} while (i <= rows3);

File: segitiga full.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
   
    int rows = 5;
    for(int i = 1; i <= rows; i++) {
        for(int j = 1; j <= rows - i; j++) {
            cout << " ";
        }
        for(int k = 1; k <= 2 * i - 1; k++) {
            cout << "*";
        }
        cout << endl;
    }

    
    cout << endl;

    
    int rows2 = 5;
    while (rows2 >= 1) {
        cout << string(rows2, '*') << endl;
        rows2--;
    }

    
    cout << endl;

    int rows3 = 5;
    int i = 1;
    do {
        cout << string(rows3 - i, ' ') << string(i, '*') << endl;
        i++;
    } while (i <= rows3);

    return 0;
}